src/black/handle_ipynb_magics.py

```diff
@@ -37,20 +37,13 @@
         "ESCAPED_NL",
     )
 )
-NON_PYTHON_CELL_MAGICS = frozenset(
+PYTHON_CELL_MAGICS = frozenset(
     (
-        "bash",
-        "html",
-        "javascript",
-        "js",
-        "latex",
-        "markdown",
-        "perl",
-        "ruby",
-        "script",
-        "sh",
-        "svg",
-        "writefile",
+        "capture",
+        "prun",
+        "pypy",
+        "time",
+        "timeit",
```

Comment:
I think `python` and `python3` should also be added?

Reply:
Yes my bad -- adding them, as well as `python2`.